Library/Homebrew/utils/formatter.rb

```diff
@@ -0,0 +1,46 @@
+require "utils/tty"
+
+module Formatter
+  module_function
+
+  def arrow(string, color: nil)
+    prefix("==>", string, color)
+  end
+
+  def headline(string, color: nil)
+    arrow("#{Tty.bold}#{string}#{Tty.reset}", color: color)
+  end
+
+  def success(string, label: nil)
+    label(label, string, :green)
+  end
+
+  def warning(string, label: nil)
+    label(label, string, :yellow)
+  end
+
+  def error(string, label: nil)
+    label(label, string, :red)
+  end
+
+  def url(string)
+    "#{Tty.blue.underline}#{string}#{Tty.reset}"
```

Comment:
Would personally prefer to stick with the underline only (no color), but I don't feel very strongly about this.